### ISSUE TITLE
Importing okra-memory in the browser results in class circular dependency error

### ISSUE THREAD

**Issue by cristiano-belloni** [canvasxyz/okra-js#1]
See reproduction here: https://codesandbox.io/p/sandbox/blissful-sanne-x3tx83?file=%2Fsrc%2FApp.tsx%3A7%2C18

This results in this error (you can open the app in an external window at this url: https://x3tx83-5173.csb.app/): 

```
Uncaught TypeError: Class extends value undefined is not a constructor or null
    at node_modules/abstract-level/abstract-level.js
```

From a brief search, it looks like it's due to a class circular dependency.

**Comment by joeltg** [comment]
It looks to me like it might actually be the `events` package, which [`memory-level`](https://github.com/Level/memory-level) uses but doesn't have as a dependency. Can you try just adding `"events": "^3.3.0"` to your package.json dependencies and seeing if that works?

**Comment by cristiano-belloni** [comment]
> It looks to me like it might actually be the `events` package, which [`memory-level`](https://github.com/Level/memory-level) uses but doesn't have as a dependency. Can you try just adding `"events": "^3.3.0"` to your package.json dependencies and seeing if that works?

That works. I don't see `memory-level` using `events` though (this is [its only source file](https://github.com/Level/memory-level/blob/main/index.js)). I suppose it's one of its dependencies?

**Comment by joeltg** [comment]
Yeah, `memory-level` extends `abstract-level` which extends `EventEmitter` [here](https://github.com/Level/abstract-level/blob/main/abstract-level.js#L5). `events` is a native module in NodeJS, so there's a gray area around who's responsible for providing the polyfill in the browsers - some bundlers do it automatically, but not Vite.

**Comment by joeltg** [comment]
I just published `v0.2.1` which adds `events` as a direct dependency.

**Comment by cristiano-belloni** [comment]
> I just published `v0.2.1` which adds `events` as a direct dependency.

Thanks!
